package.json

```diff
@@ -59,56 +59,59 @@
     "test:baseUrl": "yarn build:website:baseUrl && yarn serve:website:baseUrl",
     "lock:update": "npx yarn-deduplicate"
   },
+  "dependencies": {
+    "babel-plugin-dynamic-import-node": "2.3.0",
+    "unified": "^9.2.2"
+  },
```

Comment:
I don't understand why this is needed here

Reply:
Because of the wrong hoisting order. We need the right versions of `unified`. See https://github.com/facebook/docusaurus/pull/6476#issuecomment-1023094732

Reply:
🤪 
does it change anything to move this as devDependency instead?

Reply:
Err, mostly to highlight that it's not there to actually do anything, but just to re-order hoisting. The problem is we don't have comments in package.json so you have to do weird stuff to give something semantics